airbyte-integrations/bases/debezium-v1-9-2/src/main/java/io/airbyte/integrations/debezium/internals/MySQLDateTimeConverter.java

```diff
@@ -40,24 +39,9 @@ public void configure(final Properties props) {}
   public void converterFor(final RelationalColumn field, final ConverterRegistration<SchemaBuilder> registration) {
     if (Arrays.stream(DATE_TYPES).anyMatch(s -> s.equalsIgnoreCase(field.typeName()))) {
       registerDate(field, registration);
-    } else if (Arrays.stream(TEXT_TYPES).anyMatch(s -> s.equalsIgnoreCase(field.typeName()))) {
```

Comment:
The new debezium version handles string values as expected so our custom logic is no required anymore